src/components/results/TableByUser.tsx

```diff
@@ -200,51 +241,74 @@ const TableByUser = ({ user, questions, responses, handleQuestionClicked }) => {
               </TableRow>
             </TableHead>
             <TableBody data-cy={buildTableByUserTableBodyCy(user.id)}>
-              {questionNames?.sort(getComparator(order)).map((qName) => (
-                <TableRow
-                  key={qName}
-                  sx={{ '&:last-child td, &:last-child th': { border: 0 } }}
-                  data-cy={TABLE_BY_USER_ENTRY_CY}
-                >
-                  <TableCell
-                    component="th"
-                    scope="row"
-                    align="left"
-                    sx={{ cursor: 'pointer' }}
-                    onClick={() =>
-                      handleQuestionClicked(
-                        questionByName.get(qName).first().data.questionId
-                      )
-                    }
-                    data-cy={TABLE_BY_USER_QUESTION_NAME_HEADER_CY}
+              {questionNames
+                ?.sort(getComparator({ order, comp: stringComparator }))
```

Comment:
Sorting does not return any value (it's in place), so here you display nothing?

Reply:
In fact, the sort method return a reference of the sorted array.

```
Sorts an array in place. This method mutates the array and returns a reference to the same array.
```